src/StatusQ/Controls/StatusButton.qml

```diff
@@ -5,13 +5,18 @@ import StatusQ.Core.Theme 0.1
 StatusBaseButton {
     id: statusButton
 
-    normalColor: type === StatusBaseButton.Type.Normal ? Theme.palette.primaryColor3
-                                                       : Theme.palette.dangerColor3
-    hoverColor:  type === StatusBaseButton.Type.Normal ? Theme.palette.primaryColor2
-                                                       : Theme.palette.dangerColor2
+    normalColor: type === StatusBaseButton.Type.Primary ? Theme.palette.primaryColor1 :
+                                                          type === StatusBaseButton.Type.Normal ? Theme.palette.primaryColor3
+                                                                                                : Theme.palette.dangerColor3
+    hoverColor: type === StatusBaseButton.Type.Primary ? Theme.palette.hoverColor(normalColor) :
+                                                         type === StatusBaseButton.Type.Normal ? Theme.palette.primaryColor2
+                                                                                               : Theme.palette.dangerColor2
     disaledColor: Theme.palette.baseColor2
```

Comment:
Not related to this PR but seen this typo:

```suggestion
    disabledColor: Theme.palette.baseColor2
```

Reply:
Existing code and I think @alexandraB99 already fixed it in another PR